slp2mp4/installer.py

```diff
@@ -32,6 +34,19 @@ def recursive_overwrite(src, dest, ignore=None):
     else:
         shutil.copyfile(src, dest)
 
+
+def patch_dolphin_sys_game_settings():
+    # Remove efb_scale field. This allows selection of resolution options from GFX.ini.
+    gal_ini = os.path.join(FM_WIN_FOLDER, "Sys", "GameSettings", "GAL.ini")
```

Comment:
This breaks Linux compatibility because FM_WIN_FOLDER is hardcoded here :'(
I think the correct solution here is to patch GAL.ini on the fly, using whatever dolphin directory is specified in config.json.
Ideally, it patches it at the start of execution, and then un-patches it after, so it doesn't mess up people's playback settings...but this could still go go wrong... hmm it's a bit annoying tbh
Maybe copying the whole playback directory into slp2mp4, and running dolphin from there is better.

Reply:
Aw I missed that I had selected that hardcoded windows path instead. Thanks for catching that. I'll switch it to patch from the config entry.

The most annoying part is that if you control-C or otherwise exit the python script we don't have a mechanism in place right now to catch it an unpatch this to exit gracefully (though we should at some point). This is why I had to create that 'clean()' function to run at startup; exiting early would not remove the temporary files.

For this specific patch, I'm not too concerned about it messing up people's playback settings. It just means it will be taking the EFBScale configuration from GFX.ini by default.